python/src/ir.cc

```diff
@@ -1616,10 +1616,16 @@ void init_triton_ir(py::module &&m) {
 }
 
 void init_triton_env_vars(py::module &m) {
-  m.def("get_env_vars", []() -> std::map<std::string, bool> {
-    std::map<std::string, bool> envVars;
+  m.def("get_env_vars", []() -> std::map<std::string, std::string> {
+    std::map<std::string, std::string> envVars;
     for (const auto &envVar : ENV_VARS) {
-      envVars[envVar] = triton::tools::getBoolEnv(envVar);
+      const char *s = std::getenv(envVar.c_str());
+      if (!s)
+        envVars[envVar] = "false";
+      else if (triton::tools::isBoolEnv(envVar))
+        envVars[envVar] = triton::tools::getBoolEnv(envVar) ? "true" : "false";
```

Comment:
Can we refactor `isBoolEnv` to take in `s` and return an optional or similar to avoid calling `getBoolEnv`?